app/Providers/AuthServiceProvider.php

```diff
@@ -25,6 +32,30 @@ public function boot()
     {
         $this->registerPolicies();
 
-        //
+        $this->registerAuthGuard();
+    }
+
+    /**
+     * @return void
+     */
+    private function registerAuthGuard(): void
+    {
+        Auth::viaRequest('jwt', function ($request) {
+            $token = $request->bearerToken();
+            abort_if(! $token, 401, 'Token not provided.');
```

Comment:
This is a fantastic case for `abort_unless($token, 401, 'Token not provided.')`.